funnel/views/account.py

```diff
@@ -155,6 +156,17 @@ def recent_organization_memberships(
     )
 
 
+@User.views('avatar_color_code', cached_property=True)
+def avatar_color_code(obj):
+    # Return an int from 1 to avatar_color_code from the initials of the given string
+    initial = initials(obj.title)
+    parts = initial.split()
+    string_total = ord(parts[0][0])
+    if len(parts) > 1:
+        string_total += ord(parts[0][1])
+    return string_total % no_avatar_colours or no_avatar_colours
```

Comment:
This is slightly more efficient if we reproduce the Baseframe filter here and replace the split+join+split with a single split:

```python
    if obj.title:
        parts = obj.title.split()
        if len(parts) > 1:
            total = ord(parts[0][0]) + ord(parts[-1][0])
        else:
            total = ord(parts[0][0])
    else:
        total = 0
    return total % no_avatar_colours or no_avatar_colours
```

Reply:
This code effectively takes a modulus from 0 to 5, and then replaces 0 with 6 to get a 1-6 scale. I think it will be cleaner to read if we use 0-5 scale everywhere including in the CSS and JavaScript version.